scripts/apps/search/components/MarkedDesksList.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {
+    closeActionsMenu
+} from 'apps/search/helpers';
+
+export class MarkedDesksList extends React.Component {
```

Comment:
The "all other JSX files I committed yesterday" are not new. They are coming from the "old" `ItemList.js` which I've split up into multiple files as part of a refactor.

New components that go into the codebase should be documented.

While I would love to spend time and document all of the old components too, that would probably take me many months of work. These requirements are part of our contribution guidelines and I don't understand what the problem is.